checkup.go

```diff
@@ -275,6 +277,15 @@ func (c *Checkup) UnmarshalJSON(b []byte) error {
 				return err
 			}
 			c.Checkers = append(c.Checkers, checker)
+		case "tcp":
+			var checker TCPChecker
+			// Default timeout
+			checker.Timeout = 1 * time.Second
```

Comment:
At first, I did was you suggested. But then, I thought this way would make it easier for anyone actually looking for `where do these guys set the default timeout`, to find it.